lib/node_modules/@stdlib/number/float32/base/significand/lib/native.js

```diff
@@ -0,0 +1,53 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Return an integer corresponding to the significand of a single-precision floating-point number.
+*
+* @private
+* @param {number} x - input value
+* @returns {integer32} - significand
+*
+* @example
+* var toFloat32 = require( '@stdlib/number/float64/base/to-float32' );
+* var s = significand( toFloat32( 3.14e34 ) ); // => 10000011000010001110111
```

Comment:
```suggestion
*
* var s = significand( toFloat32( 3.14e34 ) ); // => 10000011000010001110111
```

We typically prefer line breaks separating the list of imports from the example logic.